docs/install.rst

```diff
@@ -1,56 +1,85 @@
 Installing Pelican
 ##################
 
-Pelican currently runs best on |min_python|; earlier versions of Python are not supported.
+Pelican currently runs best on Python |min_python|; earlier versions of Python are not supported.
 
-You can install Pelican via several different methods. The simplest is via Pip_::
+Once Pelican is installed, you can run ``pelican --help`` to see basic usage
+options. For more detail, refer to the :doc:`Publish<publish>` section.
+
+You can install Pelican via several different methods.
+
+**Recommended method:** `Pip <https://pip.pypa.io/>`_ User Install
```

Comment:
I think it might be worth making `uv` the recommended method; it solves _so_ many of the possible permutations (python versions, dependency locks, etc).

If we do that, we can rewrite this with a whole bunch fewer variations:

- Basic install: 
    1. install uv
    2. `uv tool install pelican`

- Optional packages: 
    - `uv tool install 'pelican[markdown]'`

That gets rid of, like, all of this :D